builders/go/pkg/build.go

```diff
@@ -179,6 +210,9 @@ func (b *GoBuild) generateCommandEnvVariables() ([]string, error) {
 		env = append(env, fmt.Sprintf("%s=%s", k, v))
 	}
 
+	// Always add the current working directory.
+	env = append(env, fmt.Sprintf("PWD=%s", os.Getenv("PWD")))
```

Comment:
Instead of recording the PWD as env variable, we could augment our provenance [to:](https://github.com/slsa-framework/slsa-github-generator-go/blob/main/builders/go/pkg/provenance.go#L40-L43) to have an additional `WorkingDir` field.

Thoughts?

Reply:
@ianlewis Wdut?

Reply:
> @ianlewis Wdut?

Sorry, did you change something? I added a comment earlier that I think including something like `WorkingDir` in the provenance is better because it's more explicit.